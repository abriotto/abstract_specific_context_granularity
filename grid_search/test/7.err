  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s] 10%|##################8                                                                                                                                                                        | 10/99 [00:00<00:01, 79.55it/s] 18%|##################################                                                                                                                                                         | 18/99 [00:00<00:01, 70.03it/s] 28%|####################################################8                                                                                                                                      | 28/99 [00:00<00:00, 74.36it/s] 38%|#######################################################################7                                                                                                                   | 38/99 [00:00<00:00, 80.35it/s] 47%|########################################################################################7                                                                                                  | 47/99 [00:00<00:00, 73.98it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:00<00:00, 69.00it/s] 63%|#####################################################################################################################1                                                                     | 62/99 [00:00<00:00, 64.71it/s] 70%|##################################################################################################################################3                                                        | 69/99 [00:01<00:00, 61.89it/s] 79%|###################################################################################################################################################3                                       | 78/99 [00:01<00:00, 67.46it/s] 86%|################################################################################################################################################################5                          | 85/99 [00:01<00:00, 66.43it/s] 93%|#############################################################################################################################################################################7             | 92/99 [00:01<00:00, 65.62it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 68.63it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 24%|#############################################1                                                                                                                                              | 6/25 [00:00<00:00, 57.54it/s] 48%|#########################################################################################7                                                                                                 | 12/25 [00:00<00:00, 55.52it/s] 72%|######################################################################################################################################6                                                    | 18/25 [00:00<00:00, 55.09it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 55.32it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 55.47it/s]
