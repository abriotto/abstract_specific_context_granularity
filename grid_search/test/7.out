# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6997891664505005, "acc": 0.5060340762138367, "length": 3.417095422744751, "mode": "train", "epoch": 1}
{"loss": 0.6938185095787048, "acc": 0.5188323855400085, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6893563270568848, "acc": 0.5345062613487244, "length": 2.2337918281555176, "mode": "train", "epoch": 2}
{"loss": 0.6949468851089478, "acc": 0.5141243934631348, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6772243976593018, "acc": 0.5808491706848145, "length": 2.062297821044922, "mode": "train", "epoch": 3}
{"loss": 0.669607400894165, "acc": 0.6131000518798828, "length": 1.6779661178588867, "mode": "test", "epoch": 3}
{"loss": 0.6339748501777649, "acc": 0.6626269817352295, "length": 2.39621901512146, "mode": "train", "epoch": 4}
{"loss": 0.590553879737854, "acc": 0.7107168436050415, "length": 2.508474588394165, "mode": "test", "epoch": 4}
{"loss": 0.5898871421813965, "acc": 0.6826193928718567, "length": 2.6129026412963867, "mode": "train", "epoch": 5}
{"loss": 0.5640258193016052, "acc": 0.7082485556602478, "length": 2.762711763381958, "mode": "test", "epoch": 5}
{"loss": 0.5476211309432983, "acc": 0.7151710987091064, "length": 2.9271392822265625, "mode": "train", "epoch": 6}
{"loss": 0.5297580361366272, "acc": 0.7643129825592041, "length": 3.1186439990997314, "mode": "test", "epoch": 6}
{"loss": 0.5144587755203247, "acc": 0.7542464733123779, "length": 3.0994017124176025, "mode": "train", "epoch": 7}
{"loss": 0.5140877366065979, "acc": 0.7624124884605408, "length": 3.3559322357177734, "mode": "test", "epoch": 7}
{"loss": 0.49770551919937134, "acc": 0.7736228704452515, "length": 3.1547954082489014, "mode": "train", "epoch": 8}
{"loss": 0.5174795985221863, "acc": 0.7622480392456055, "length": 3.254237174987793, "mode": "test", "epoch": 8}
{"loss": 0.4761026203632355, "acc": 0.789498507976532, "length": 3.215898275375366, "mode": "train", "epoch": 9}
{"loss": 0.4957601726055145, "acc": 0.7932435870170593, "length": 3.4406778812408447, "mode": "test", "epoch": 9}
{"loss": 0.46748510003089905, "acc": 0.7942953705787659, "length": 3.272739887237549, "mode": "train", "epoch": 10}
{"loss": 0.5127149820327759, "acc": 0.7844695448875427, "length": 3.254237174987793, "mode": "test", "epoch": 10}
{"loss": 0.4566541314125061, "acc": 0.813040554523468, "length": 3.247438907623291, "mode": "train", "epoch": 11}
{"loss": 0.4744516611099243, "acc": 0.7925615906715393, "length": 3.4237287044525146, "mode": "test", "epoch": 11}
{"loss": 0.42892658710479736, "acc": 0.8158736228942871, "length": 3.318873643875122, "mode": "train", "epoch": 12}
{"loss": 0.4714817702770233, "acc": 0.793086051940918, "length": 3.361581802368164, "mode": "test", "epoch": 12}
{"loss": 0.41325604915618896, "acc": 0.8216161131858826, "length": 3.3238420486450195, "mode": "train", "epoch": 13}
{"loss": 0.4711892604827881, "acc": 0.792016327381134, "length": 3.378530979156494, "mode": "test", "epoch": 13}
{"loss": 0.4042680561542511, "acc": 0.8257685303688049, "length": 3.3517258167266846, "mode": "train", "epoch": 14}
{"loss": 0.43940919637680054, "acc": 0.8123788833618164, "length": 3.6101694107055664, "mode": "test", "epoch": 14}
{"loss": 0.39033910632133484, "acc": 0.8237576484680176, "length": 3.500217914581299, "mode": "train", "epoch": 15}
{"loss": 0.42798376083374023, "acc": 0.817864179611206, "length": 3.5875706672668457, "mode": "test", "epoch": 15}
{"loss": 0.3737390637397766, "acc": 0.8449472188949585, "length": 3.4902312755584717, "mode": "train", "epoch": 16}
{"loss": 0.4475894868373871, "acc": 0.8313550353050232, "length": 3.728813648223877, "mode": "test", "epoch": 16}
{"loss": 0.3712065517902374, "acc": 0.8368343710899353, "length": 3.666625499725342, "mode": "train", "epoch": 17}
{"loss": 0.45094919204711914, "acc": 0.8123858571052551, "length": 3.728813648223877, "mode": "test", "epoch": 17}
{"loss": 0.3649083971977234, "acc": 0.8438937067985535, "length": 3.6265578269958496, "mode": "train", "epoch": 18}
{"loss": 0.4390495717525482, "acc": 0.8102775812149048, "length": 3.728813648223877, "mode": "test", "epoch": 18}
{"loss": 0.34515488147735596, "acc": 0.8482540249824524, "length": 3.730067729949951, "mode": "train", "epoch": 19}
{"loss": 0.4583500623703003, "acc": 0.8311852216720581, "length": 3.8983051776885986, "mode": "test", "epoch": 19}
{"loss": 0.35220572352409363, "acc": 0.8511232733726501, "length": 3.8127479553222656, "mode": "train", "epoch": 20}
{"loss": 0.4525838792324066, "acc": 0.8241335153579712, "length": 3.870056390762329, "mode": "test", "epoch": 20}
{"loss": 0.34316757321357727, "acc": 0.8580557703971863, "length": 3.750192642211914, "mode": "train", "epoch": 21}
{"loss": 0.4329783320426941, "acc": 0.8266831636428833, "length": 3.819209098815918, "mode": "test", "epoch": 21}
{"loss": 0.3349916338920593, "acc": 0.8580005168914795, "length": 3.7700462341308594, "mode": "train", "epoch": 22}
{"loss": 0.4354144036769867, "acc": 0.8287724256515503, "length": 3.836158275604248, "mode": "test", "epoch": 22}
{"loss": 0.3258083164691925, "acc": 0.8600296974182129, "length": 3.7924962043762207, "mode": "train", "epoch": 23}
{"loss": 0.4456993043422699, "acc": 0.8360112309455872, "length": 3.8418078422546387, "mode": "test", "epoch": 23}
{"loss": 0.3220524787902832, "acc": 0.8674164414405823, "length": 3.7958645820617676, "mode": "train", "epoch": 24}
{"loss": 0.4286569356918335, "acc": 0.8426907062530518, "length": 3.8813560009002686, "mode": "test", "epoch": 24}
{"loss": 0.31252866983413696, "acc": 0.8711696863174438, "length": 3.8276920318603516, "mode": "train", "epoch": 25}
{"loss": 0.46529245376586914, "acc": 0.8219162821769714, "length": 3.9322032928466797, "mode": "test", "epoch": 25}
{"loss": 0.2999740540981293, "acc": 0.8743395805358887, "length": 3.890449285507202, "mode": "train", "epoch": 26}
{"loss": 0.4216914176940918, "acc": 0.8385711312294006, "length": 3.9152543544769287, "mode": "test", "epoch": 26}
{"loss": 0.30690017342567444, "acc": 0.8767210841178894, "length": 3.858920097351074, "mode": "train", "epoch": 27}
{"loss": 0.41083580255508423, "acc": 0.8397222757339478, "length": 3.853107452392578, "mode": "test", "epoch": 27}
{"loss": 0.2915203273296356, "acc": 0.8791628479957581, "length": 3.8112168312072754, "mode": "train", "epoch": 28}
{"loss": 0.42806321382522583, "acc": 0.8315869569778442, "length": 3.8418078422546387, "mode": "test", "epoch": 28}
{"loss": 0.28437700867652893, "acc": 0.8775629997253418, "length": 3.862812042236328, "mode": "train", "epoch": 29}
{"loss": 0.3944464325904846, "acc": 0.847405731678009, "length": 3.8983051776885986, "mode": "test", "epoch": 29}
{"loss": 0.2851099669933319, "acc": 0.8795153498649597, "length": 3.891432762145996, "mode": "train", "epoch": 30}
{"loss": 0.3965427875518799, "acc": 0.8456627726554871, "length": 3.9322032928466797, "mode": "test", "epoch": 30}
{"loss": 0.2798439860343933, "acc": 0.8838809132575989, "length": 3.899688720703125, "mode": "train", "epoch": 31}
{"loss": 0.39028555154800415, "acc": 0.8501353859901428, "length": 3.909604549407959, "mode": "test", "epoch": 31}
{"loss": 0.27257639169692993, "acc": 0.8899100422859192, "length": 3.8939707279205322, "mode": "train", "epoch": 32}
{"loss": 0.3668580651283264, "acc": 0.8609934449195862, "length": 3.8983051776885986, "mode": "test", "epoch": 32}
{"loss": 0.26132771372795105, "acc": 0.8919016718864441, "length": 3.8882484436035156, "mode": "train", "epoch": 33}
{"loss": 0.40041932463645935, "acc": 0.8591691255569458, "length": 3.9152543544769287, "mode": "test", "epoch": 33}
{"loss": 0.2659374177455902, "acc": 0.8912532925605774, "length": 3.9094130992889404, "mode": "train", "epoch": 34}
{"loss": 0.41173601150512695, "acc": 0.845839262008667, "length": 3.9378530979156494, "mode": "test", "epoch": 34}
{"loss": 0.2684437334537506, "acc": 0.8851054906845093, "length": 3.953094005584717, "mode": "train", "epoch": 35}
{"loss": 0.3824044167995453, "acc": 0.8593456149101257, "length": 3.9491524696350098, "mode": "test", "epoch": 35}
{"loss": 0.24480710923671722, "acc": 0.9003356695175171, "length": 3.941746950149536, "mode": "train", "epoch": 36}
{"loss": 0.36526888608932495, "acc": 0.8705567717552185, "length": 3.9322032928466797, "mode": "test", "epoch": 36}
{"loss": 0.243953675031662, "acc": 0.9015583992004395, "length": 3.9249720573425293, "mode": "train", "epoch": 37}
{"loss": 0.38490259647369385, "acc": 0.8678202629089355, "length": 3.943502902984619, "mode": "test", "epoch": 37}
{"loss": 0.25201496481895447, "acc": 0.894433856010437, "length": 3.945617198944092, "mode": "train", "epoch": 38}
{"loss": 0.37871718406677246, "acc": 0.8596399426460266, "length": 3.9717514514923096, "mode": "test", "epoch": 38}
{"loss": 0.24056047201156616, "acc": 0.9017939567565918, "length": 3.959829568862915, "mode": "train", "epoch": 39}
{"loss": 0.4216609001159668, "acc": 0.8464277982711792, "length": 3.9943501949310303, "mode": "test", "epoch": 39}
{"loss": 0.2523828148841858, "acc": 0.9015030264854431, "length": 3.9499974250793457, "mode": "train", "epoch": 40}
{"loss": 0.3869830071926117, "acc": 0.8558440208435059, "length": 3.9943501949310303, "mode": "test", "epoch": 40}
{"loss": 0.23388053476810455, "acc": 0.9074370861053467, "length": 3.951868772506714, "mode": "train", "epoch": 41}
{"loss": 0.3699667453765869, "acc": 0.8668785095214844, "length": 3.9774010181427, "mode": "test", "epoch": 41}
{"loss": 0.230759397149086, "acc": 0.9108477830886841, "length": 3.946786642074585, "mode": "train", "epoch": 42}
{"loss": 0.38366955518722534, "acc": 0.8615232110023499, "length": 3.9774010181427, "mode": "test", "epoch": 42}
{"loss": 0.22500558197498322, "acc": 0.9110445976257324, "length": 3.9589016437530518, "mode": "train", "epoch": 43}
{"loss": 0.3857444226741791, "acc": 0.8733816146850586, "length": 3.9717514514923096, "mode": "test", "epoch": 43}
{"loss": 0.22106432914733887, "acc": 0.9079549312591553, "length": 3.9666502475738525, "mode": "train", "epoch": 44}
{"loss": 0.3703044652938843, "acc": 0.8738229870796204, "length": 3.9774010181427, "mode": "test", "epoch": 44}
{"loss": 0.20615805685520172, "acc": 0.9183344841003418, "length": 3.9631569385528564, "mode": "train", "epoch": 45}
{"loss": 0.366560161113739, "acc": 0.8766478896141052, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.2122422605752945, "acc": 0.919647216796875, "length": 3.968435525894165, "mode": "train", "epoch": 46}
{"loss": 0.3963202238082886, "acc": 0.8559321761131287, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.2158418595790863, "acc": 0.9144929647445679, "length": 3.971848964691162, "mode": "train", "epoch": 47}
{"loss": 0.374350368976593, "acc": 0.8644068837165833, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.21075284481048584, "acc": 0.9215736389160156, "length": 3.9907586574554443, "mode": "train", "epoch": 48}
{"loss": 0.41297444701194763, "acc": 0.8634651899337769, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.2145477831363678, "acc": 0.913293182849884, "length": 3.985966444015503, "mode": "train", "epoch": 49}
{"loss": 0.4063762128353119, "acc": 0.8625235557556152, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.21518443524837494, "acc": 0.9143402576446533, "length": 3.9814255237579346, "mode": "train", "epoch": 50}
{"loss": 0.3996967673301697, "acc": 0.8757061958312988, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.21640169620513916, "acc": 0.9185884594917297, "length": 3.9877712726593018, "mode": "train", "epoch": 51}
{"loss": 0.38053596019744873, "acc": 0.8738231062889099, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.20676332712173462, "acc": 0.921664834022522, "length": 3.986391544342041, "mode": "train", "epoch": 52}
{"loss": 0.42099854350090027, "acc": 0.8672317266464233, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.1998208463191986, "acc": 0.9229403734207153, "length": 3.9881882667541504, "mode": "train", "epoch": 53}
{"loss": 0.41053444147109985, "acc": 0.8587571382522583, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.20519554615020752, "acc": 0.9219465255737305, "length": 3.993516445159912, "mode": "train", "epoch": 54}
{"loss": 0.3702468276023865, "acc": 0.8681733012199402, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.1842428296804428, "acc": 0.9282282590866089, "length": 3.991514205932617, "mode": "train", "epoch": 55}
{"loss": 0.375679612159729, "acc": 0.8813559412956238, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.1833425611257553, "acc": 0.9303217530250549, "length": 3.9931275844573975, "mode": "train", "epoch": 56}
{"loss": 0.39111313223838806, "acc": 0.8578155040740967, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.17455826699733734, "acc": 0.9332552552223206, "length": 3.9932644367218018, "mode": "train", "epoch": 57}
{"loss": 0.3661800026893616, "acc": 0.8691149353981018, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.1802206039428711, "acc": 0.9331907033920288, "length": 3.9920432567596436, "mode": "train", "epoch": 58}
{"loss": 0.40632039308547974, "acc": 0.87664794921875, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.1960201859474182, "acc": 0.9175043702125549, "length": 3.9958043098449707, "mode": "train", "epoch": 59}
{"loss": 0.3698490262031555, "acc": 0.8804143667221069, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.20251226425170898, "acc": 0.9223334789276123, "length": 3.9950265884399414, "mode": "train", "epoch": 60}
{"loss": 0.388133704662323, "acc": 0.8766478896141052, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.19555726647377014, "acc": 0.9261224865913391, "length": 3.995960235595703, "mode": "train", "epoch": 61}
{"loss": 0.3986823558807373, "acc": 0.8691149950027466, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.17085690796375275, "acc": 0.9347127079963684, "length": 3.991295576095581, "mode": "train", "epoch": 62}
{"loss": 0.35784125328063965, "acc": 0.8700565695762634, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.15918943285942078, "acc": 0.9377670288085938, "length": 3.986255168914795, "mode": "train", "epoch": 63}
{"loss": 0.3700889050960541, "acc": 0.8813560009002686, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.16419075429439545, "acc": 0.9397754073143005, "length": 3.9883201122283936, "mode": "train", "epoch": 64}
{"loss": 0.40380579233169556, "acc": 0.8775895237922668, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.16363823413848877, "acc": 0.9408941864967346, "length": 3.9980084896087646, "mode": "train", "epoch": 65}
{"loss": 0.39953383803367615, "acc": 0.8662900924682617, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.16932527720928192, "acc": 0.9396616816520691, "length": 3.995450019836426, "mode": "train", "epoch": 66}
{"loss": 0.3963002562522888, "acc": 0.882297694683075, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.1665624976158142, "acc": 0.9398165941238403, "length": 3.993065595626831, "mode": "train", "epoch": 67}
{"loss": 0.4093571603298187, "acc": 0.8728813529014587, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.17092660069465637, "acc": 0.9339567422866821, "length": 3.995307683944702, "mode": "train", "epoch": 68}
{"loss": 0.35822516679763794, "acc": 0.8851223587989807, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.16256888210773468, "acc": 0.9391922950744629, "length": 3.987799882888794, "mode": "train", "epoch": 69}
{"loss": 0.39683887362480164, "acc": 0.8822975158691406, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.16443900763988495, "acc": 0.935845136642456, "length": 3.9920003414154053, "mode": "train", "epoch": 70}
{"loss": 0.3582337498664856, "acc": 0.883239209651947, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.14810052514076233, "acc": 0.9450192451477051, "length": 3.9981656074523926, "mode": "train", "epoch": 71}
{"loss": 0.3609696626663208, "acc": 0.8898305892944336, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.1503533273935318, "acc": 0.9446983337402344, "length": 3.9971206188201904, "mode": "train", "epoch": 72}
{"loss": 0.3837233781814575, "acc": 0.8766477704048157, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.14617030322551727, "acc": 0.9431678056716919, "length": 3.999361038208008, "mode": "train", "epoch": 73}
{"loss": 0.3712882399559021, "acc": 0.8719397783279419, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.14159026741981506, "acc": 0.9520137906074524, "length": 3.997938632965088, "mode": "train", "epoch": 74}
{"loss": 0.38209009170532227, "acc": 0.8728813529014587, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.14754793047904968, "acc": 0.9459448456764221, "length": 3.995657205581665, "mode": "train", "epoch": 75}
{"loss": 0.3941551446914673, "acc": 0.8728813529014587, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.15603840351104736, "acc": 0.9453408122062683, "length": 3.9967257976531982, "mode": "train", "epoch": 76}
{"loss": 0.4079412817955017, "acc": 0.8794726133346558, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.1637474149465561, "acc": 0.9339524507522583, "length": 3.9966859817504883, "mode": "train", "epoch": 77}
{"loss": 0.4076492190361023, "acc": 0.8794727325439453, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.1575666069984436, "acc": 0.9434514045715332, "length": 3.9985837936401367, "mode": "train", "epoch": 78}
{"loss": 0.4104563593864441, "acc": 0.8898305892944336, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.14272916316986084, "acc": 0.944417417049408, "length": 3.9975457191467285, "mode": "train", "epoch": 79}
{"loss": 0.37865370512008667, "acc": 0.887005627155304, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.1395760178565979, "acc": 0.9512642025947571, "length": 3.9950616359710693, "mode": "train", "epoch": 80}
{"loss": 0.35071635246276855, "acc": 0.8926555514335632, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.14027093350887299, "acc": 0.9510236382484436, "length": 3.998682737350464, "mode": "train", "epoch": 81}
{"loss": 0.3485097289085388, "acc": 0.8917137980461121, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.13783740997314453, "acc": 0.9507234692573547, "length": 3.9990899562835693, "mode": "train", "epoch": 82}
{"loss": 0.3738710284233093, "acc": 0.8992467522621155, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.1521647870540619, "acc": 0.9479107856750488, "length": 3.999760866165161, "mode": "train", "epoch": 83}
{"loss": 0.4213365912437439, "acc": 0.8794726133346558, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.14500567317008972, "acc": 0.9510692358016968, "length": 3.999643087387085, "mode": "train", "epoch": 84}
{"loss": 0.41419386863708496, "acc": 0.8907721638679504, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.14507625997066498, "acc": 0.9475951790809631, "length": 3.9998159408569336, "mode": "train", "epoch": 85}
{"loss": 0.3866496682167053, "acc": 0.8917137980461121, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.1472136229276657, "acc": 0.9459518790245056, "length": 3.9955711364746094, "mode": "train", "epoch": 86}
{"loss": 0.36856016516685486, "acc": 0.8879472017288208, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.1390194594860077, "acc": 0.949847936630249, "length": 3.993558406829834, "mode": "train", "epoch": 87}
{"loss": 0.37850528955459595, "acc": 0.8945387601852417, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.1344716101884842, "acc": 0.9529626965522766, "length": 3.9997692108154297, "mode": "train", "epoch": 88}
{"loss": 0.3777768015861511, "acc": 0.8822975754737854, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.13511408865451813, "acc": 0.9560537338256836, "length": 3.997114658355713, "mode": "train", "epoch": 89}
{"loss": 0.42031949758529663, "acc": 0.895480215549469, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.12534913420677185, "acc": 0.9542067646980286, "length": 3.9992454051971436, "mode": "train", "epoch": 90}
{"loss": 0.3981931209564209, "acc": 0.9011300206184387, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.12250421941280365, "acc": 0.953912079334259, "length": 3.999830722808838, "mode": "train", "epoch": 91}
{"loss": 0.43414756655693054, "acc": 0.8964218497276306, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.12709122896194458, "acc": 0.9553393721580505, "length": 3.9936084747314453, "mode": "train", "epoch": 92}
{"loss": 0.42456087470054626, "acc": 0.8738229870796204, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.12346082925796509, "acc": 0.9545450806617737, "length": 3.999887228012085, "mode": "train", "epoch": 93}
{"loss": 0.4193361699581146, "acc": 0.8945385813713074, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.12352034449577332, "acc": 0.9554651975631714, "length": 3.999195098876953, "mode": "train", "epoch": 94}
{"loss": 0.41941893100738525, "acc": 0.8926553726196289, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.11803021281957626, "acc": 0.9548521041870117, "length": 3.9992592334747314, "mode": "train", "epoch": 95}
{"loss": 0.4038323760032654, "acc": 0.8860639333724976, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.12761059403419495, "acc": 0.9551727771759033, "length": 3.9998955726623535, "mode": "train", "epoch": 96}
{"loss": 0.4078277349472046, "acc": 0.8907722234725952, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.13044434785842896, "acc": 0.9498032331466675, "length": 3.9997050762176514, "mode": "train", "epoch": 97}
{"loss": 0.3938271999359131, "acc": 0.8860641121864319, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.12301014363765717, "acc": 0.9532790780067444, "length": 3.9998371601104736, "mode": "train", "epoch": 98}
{"loss": 0.34723302721977234, "acc": 0.8935970067977905, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.12193416059017181, "acc": 0.9586387276649475, "length": 3.9992852210998535, "mode": "train", "epoch": 99}
{"loss": 0.36662089824676514, "acc": 0.8935970067977905, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.12699587643146515, "acc": 0.9561281800270081, "length": 3.99845552444458, "mode": "train", "epoch": 100}
{"loss": 0.43849754333496094, "acc": 0.888888955116272, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.8427349328994751
